core/code/comm.js

```diff
@@ -0,0 +1,780 @@
+/**
+ * @file Namespace for comm-related functionalities.
+ * @namespace window.comm
+ */
+
+var comm = {};
+window.comm = comm;
+
+/**
+ * Holds data related to each intel channel.
+ *
+ * @memberof window.comm
+ * @type {Object}
+ */
+comm._channels = {};
+
+/**
+ * Initialize the channel data.
+ *
+ * @function comm.initChannelData
```

Comment:
```suggestion
 * @function window.comm.initChannelData
```
The same change needs to be made for all functions. Otherwise jsdoc does not see the function